.travis.yml

```diff
@@ -3,6 +3,9 @@ sudo: false
 cache: cargo
 
 rust:
+  #- nightly
+  # Override because diesel is broken under current nightly
```

Comment:
So this doesn't need a specific nightly anymore?

Reply:
Nope. the previous restriction was in place because of a diesel bug. Current nightly fixes that problem.

Reply:
For how long? ;)